### ISSUE TITLE
Update ambiguous phrasing for connecting signals to a Callable

### ISSUE THREAD

**Issue by beckymorley** [godotengine/godot-docs#9959]
**Your Godot version:** 4.2

**Issue description:** 
The highlighted wording below had me stumped for some time:
<img width="622" alt="image" src="https://github.com/user-attachments/assets/9bed5c04-27e6-4605-9b37-53a8e3071e59">

"A signal can only be connected once to the same Callable."

I read it as "A signal can only be connected once to _each unique_ callable" (as in connecting to both NodeClass1._do_node1_things() and NodeClass2._do_node2_things() would be acceptable) as opposed to what I now understand it's real meaning to be which is "A signal can only be connected once to _one_ Callable". With my erroneous interpretation, the next sentence "If the signal is already connected..." still makes sense to me, so the pin didn't drop even after I searched for why I was getting a "signal already connected" error. 

I think using the word "same" introduces the ambiguity, so changing this to say "one" would probably be enough to clear up the meaning. Maybe also a brief clarification that the signal needs to be disconnected from one callable in order to connect to another, just to leave absolutely no room for interpretation? 

I hope it doesn't sound like I'm nit picking, but it really had me confused for a few days and it wasn't clear that my assumption was wrong until it clicked, otherwise I probably would've asked somewhere.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/4.2/classes/class_signal.html

**Comment by AThousandShips** [comment]
> A signal can only be connected once to one Callable

This is wrong, a signal can be connected to many callables, not just once, it just can't be connected to the same callable, with the same one meaning the same object and method
